src/io/decagames/rotmg/supportCampaign/tab/tiers/preview/TiersPreview.as

```diff
@@ -48,24 +59,27 @@ package io.decagames.rotmg.supportCampaign.tab.tiers.preview
 			addChild(this._leftArrow);
 		}
 
-		public function showTier(_arg_1:int, _arg_2:int, _arg_3:int):void
+		public function showTier(_arg_1:int, _arg_2:int, _arg_3:int, _arg_4:String):void
 		{
-			var _local_5:* = _arg_1;
-			var _local_4:* = _arg_2;
-			var _local_6:* = _arg_3;
-			if (((this.background) && (this.background.parent)))
-			{
-				removeChild(this.background);
-			}
-			try
+			/*var _local_5:*/
+			this._tier = _arg_1; //_tier
```

Comment:
It's okay to rename default "_local_" variables to human-readable variants, but no need to comment old variant.